### DISCUSSION TITLE
[Submission format][Hw6]

### DISCUSSION THREAD

**Discussion by kopter0** [prosyslab-classroom/cs348-information-security#747] [Q&A]
I cannot find any information on how to submit (mail or klms) and submission format (doc, txt, tar, how to correctly enumerate questions, file name). 
And once again still no information on reclaiming of hw1 P2 ㅠㅠ.

**Comment by minsukk-at-kaist-ac-kr** [comment]
I will soon send out an email regarding the submission information for HW6. 

By the way, please email me if you have any concerns about your HW1 P2. 

Min Suk
